### ISSUE TITLE
Feature vs theme

### ISSUE THREAD

**Issue by josephrbennett** [NCC-CNC/whattodo#40]
I was wondering: what is distinction between "theme" in tool 1 and "feature" in tool 2 (apart from latter being more broadly defined perhaps)?

**Comment by jeffreyhanson** [comment]
Yeah, excellent question! My understanding is that theme is used in Tool 1 to refer to a collection of features (including a collection  that contains a single feature). For example, you might have a theme of "Species", which contains the following features: "orange-winged magpie", "blue-eyed owl", "green-footed skink". Since Tool 2 only handles each feature seperately, it does not have themes. @ricschuster - is that correct?

**Comment by ricschuster** [comment]
That's correct. One thing I should note is that feature and layer are more or less used interchangeably, maybe layer even more so because that's what GIS staff is most familiar with. 

I don't think layer is the 'best' term to use, but wonder if you have other ideas than feature as well?

**Comment by jeffreyhanson** [comment]
Sorry for my slow repsonse. I just realized that the WTD tool actually does use themes (similar to WTW tool) - the only difference is that each theme can only contain a single feature in the WTD. So, I suppose they are fairly consistent in that respect.

Yeah, I think "layer" is a good way to describe the way the underlying data is organized. E.g., each theme contains one or more features (per SCP terminology, where a feature is something we want to conserve), and each feature is associated with a single GIS layer. Each weight is is also associated with a single GIS layer, and so to is each Include. I think the main difference is that themes/features/weights/include terminology convey information on the intended purpose of the data. Whereas, the GIS term "layer" is purely just about how the data are stored in files/memory. How does that sound?

**Comment by ricschuster** [comment]
No worries.

I agree with the description you provided.
@josephrbennett what do you think?
